src/trading-api/next/v2/bullish-trading-api.yml

```diff
@@ -501,6 +502,24 @@ info:
     }
     ```
     
+    ### Command for Transfer Asset 
+    `POST /trading-api/v1/command?commandType=V1TransferAsset`
+    
+    ```json
+    {
+      "timestamp": "<TIMESTAMP>",
+      "nonce": "<NONCE>",
+      "authorizer": "<AUTHORIZER>"
+      "command": {
+        "commandType": "V1TransferAsset",
+        "assetSymbol": "USD",
+        "quantity": "7.0000",
+        "fromTradingAccountId": "11123456789",
+        "toTradingAccountId": "11198765432"
+      }
+    }
+    ```
```

Comment:
Got a question.

I see that there's a pattern to give api endpoint examples like this. 

https://github.com/bullish-exchange/api-docs/pull/36/files#diff-2ac458cdd99e39457aa20ae9634da52b4748a5a299a13628baa14ada45aa3192L470-L485


===============

However, is this not duplicating the documentation here already?

https://github.com/bullish-exchange/api-docs/pull/36/files#diff-2ac458cdd99e39457aa20ae9634da52b4748a5a299a13628baa14ada45aa3192R3197-R3218

Reply:
Yes. A bit duplication. But it is more like explaining things in different ways.